code/modules/mining/ores_coins.dm

```diff
@@ -369,6 +370,55 @@
 /obj/item/weapon/coin/is_screwdriver(var/mob/user)
 	return user.a_intent == I_HURT
 
+/obj/item/weapon/coin/proc/coinflip(mob/user as mob, thrown, rigged = FALSE)
```

Comment:
`as mob` is not needed.

Reply:
copied straight from dicecode, that shit needs a serious tune-up